src/components/TF_FIGURE/index.tsx

```diff
@@ -25,7 +27,13 @@ function TF_FIGURE(props: Props) {
   } = props;
 
   const svgContainerRef = useRef<HTMLDivElement>(null);
-  const [chart_type, update_chart_type] = useState<"line" | "bar">("line");
+  const [chart_type = "line", update_chart_type] = useQueryParam<
+    string | undefined
+  >("chart_type", mainQueryParamsConfig.chart_type);
+  const valid_chart_type = chart_type === "bar" ? "bar" : "line";
+  chart_type !== valid_chart_type &&
```

Comment:
Same. Dont think we need to remove typos from url

Reply:
I concur